tv.py

```
@@ -0,0 +1,807 @@
+  if countfuckingshit == 3:
+        logging.info("edit rtmp key fail shutdown script")
+        os.system("taskkill /f /im countdriver.exe")
+        exit()
+
+def check_is_live_api(url, ffmpeg, text):
+      countshit = 0
+      while True:
+            try:
+                  print(url)
+                  streams = streamlink.streams(url)
+                  hls_stream = streams["best"]
```

Ruff (0.8.2)

Local variable `hls_stream` is assigned to but never used

Remove assignment to unused variable `hls_stream`

(F841)